src/jit/codegencommon.cpp

```diff
@@ -3891,7 +3891,7 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
 #endif // !_TARGET_ARM_
 
 #if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
-        if (regType != TYP_STRUCT)
+        if (!varTypeIsStruct(regType))
```

Comment:
I would suggest to do a source search for all occurrences of '!= TYP_STRUCT' and '== TYP_STRUCT' and audit them to make sure that we are not leaving any checks that needs to be converted to !varTypeIsStruct() or VarTypeIsStruct() respectively.

Particularly this will be useful, after a big code integration or check-in of JIT changes done recently.